storage/localstate_mgr.go

```diff
@@ -133,6 +138,7 @@ func (fsm *FileStateMgr) GetLocalState(pubKey string) (KeygenLocalState, error)
 	if err := json.Unmarshal(pt, &localState); nil != err {
 		return KeygenLocalState{}, fmt.Errorf("fail to unmarshal KeygenLocalState:%x %w", pt, err)
 	}
+	fsm.keyGen = &localState
```

Comment:
Why do we need this? A get method shouldn't modify the state of the object

Reply:
this is populating the object with data from the file if it isn't already in memory. This will prevent a read every time the local keygen data is accessed. Also, the file won't need to be on disk while the process is running.